product-guide/manual.adoc

```diff
@@ -197,6 +197,10 @@ Specific procedures for performing an installation using a local Puppet apply ar
 Conditions for installing the ANMS are a host with packages identified in <<target-host-packages>>, at least 7{nbsp}GiB of filesystem space for container image storage, and additional space for long-term data warehouse storage.
 The total amount of storage needed depends on the mission use of reports, specifically the average size and rate of reported data.
 
+Note: Usage of podman is recommended. Docker should continue to function as a drop-in replacement, however only podman deployments will be directly supported by the ANMS team.
```

Comment:
Prefer to use [AsciiDoc admonition syntax](https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/#admonition-syntax) of capitals "NOTE: " here and below.